src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -233,6 +235,10 @@ void SetParameter()
    if ( CCSN_Prob == Migration_Test )
       CCSN_Is_PostBounce = 1;
 
+// check OPT__FLAG_REGION is disabled for the migration test
+   if ( CCSN_Prob == Migration_Test  &&  OPT__FLAG_REGION )
+      Aux_Error( ERROR_INFO, "%s is enabled for %s = %s !!\n", "OPT__FLAG_REGION", "CCSN_Prob", Migration_Test );
```

Comment:
Replace "enabled" with "is not supported"

Reply:
I have replaced this.